expressEngine/lib/service/db.js

```diff
@@ -14,6 +14,9 @@ class DbOperation {
   findOne = (collection, data = {}, projection = {}) => {
     return collection.findOne(data, projection);
   }
+  findAll = (collection, {skip, limit}, projection ={}) => {
```

Comment:
Pass fetch query and use it, use parseint to convert string to number.

Reply:
Implemented parseInt